src/libraries/System.Text.Json/src/System/Text/Json/Serialization/JsonSerializerOptions.cs

```diff
@@ -594,8 +592,8 @@ public bool WriteIndented
         // Workaround https://github.com/dotnet/linker/issues/2715
         [UnconditionalSuppressMessage("AotAnalysis", "IL3050:RequiresDynamicCode",
             Justification = "Dynamic path is guarded by the runtime feature switch.")]
-        internal MemberAccessor MemberAccessorStrategy =>
-            _memberAccessorStrategy ??=
+        internal static MemberAccessor MemberAccessorStrategy =>
+            s_memberAccessorStrategy ??=
```

Comment:
Are all of these ReflectionXxAccessor types thread-safe?

Reply:
Yes, necessarily so since `JsonSerializerOptions` is also thread-safe.